.github/workflows/pkg.yml

```
@@ -33,15 +42,86 @@ jobs:
 
           # Optional: if set to true then the action don't cache or restore ~/.cache/go-build.
           # skip-build-cache: true
+
+      - name: Print lint report artifact
+        if: always()
+        run: test -f golangci-lint-report.xml && cat golangci-lint-report.xml || true
+
+      - name: Upload lint report artifact
+        if: always()
+        uses: actions/upload-artifact@3cea5372237819ed00197afe530f5a7ea3e805c8 # v3.1.0
+        with:
+          name: golangci-lint-report
+          path: golangci-lint-report.xml
+
   build-test:
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
-      - uses: actions/setup-go@v3
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Set up Go
+        uses: actions/setup-go@v3
         with:
           go-version-file: 'go.mod'
-      - name: build
+
+      - name: Build
         run: go build -v ./...
-      - name: test
+
+      - name: Unit Tests
+        run: go test -v ./... -coverpkg=./... -coverprofile=coverage.txt $1 | tee $OUTPUT_FILE
```

Is `-v` required? It spams all of the logs from every test instead of just failures, and requires devs to use tooling to find the actual failures among the noise.
```suggestion
        run: go test ./... -coverpkg=./... -coverprofile=coverage.txt $1 | tee $OUTPUT_FILE
```

---

No, it is not required. Thanks, `-v` removed.